packages/dds/merge-tree/src/test/snapshot.utils.ts

```diff
@@ -132,7 +132,7 @@ export class TestString {
 		this.client = client2;
 	}
 
-	public getSummary() {
+	public getSummary(): ISummaryTree {
```

Comment:
Explicit return types are good.
Curious if this is a change from before or tsc sees things a little different and return type was good to clarify. ?